api/src/pcapi/routes/native/v1/cookies_consent.py

```diff
@@ -0,0 +1,19 @@
+import logging
+
+from pcapi.serialization.decorator import spectree_serialize
+
+from . import blueprint
+from .serialization import cookies_consent as serializers
+
+
+logger = logging.getLogger(__name__)
+
+
+@blueprint.native_v1.route("/cookies_consent", methods=["POST"])
+@spectree_serialize(
+    on_success_status=204,
+    on_error_statuses=[400],
+    api=blueprint.api,
+)
+def cookies_consent(body: serializers.CookieConsentRequest) -> None:
+    logger.info("Cookies consent", extra=body.dict())
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❔ question:** ‏Du coup ce qu'on fait des cookies est géré dans le front ?
On aura pas besoin d'accéder à cette donnée depuis l'app ?

Reply:
les cookies sont gérés cotés frontend

le backend ne fait que logguer